Custom Axios options (request config) (#213)
SAP/cloud-sdk: docs/js/features/common/custom-request-config.mdx

> By default, the SAP Cloud SDK uses `axios` as HTTP client for executing requests.

Reviewer: ```suggestion
By default, the SAP Cloud SDK uses `[axios](https://github.com/axios/axios)` as an HTTP client backend for executing requests.
```

> In addition to basic options like `url` and `headers`, being generated by the SAP Cloud SDK under the hood, you are also allowed to provide custom request configuration, which is then used as custom `axios` options by default.

Reviewer: ```suggestion
The SAP Cloud SDK derives and configures most of the request options like `url`, `headers`, and others for you. In certain cases passing additional options to `axios` might be required. To achieve this you can provide a custom request configuration that will be passed down to `axios` and in turn, applied to your requests down with the SAP Cloud SDK.
```

> The example below demonstrates how to indicate the response data type, which is typically used when downloading a file from an endpoint.

Reviewer: ```suggestion
The example below demonstrates how to configure the response data type, which is typically used when downloading a file from an endpoint.
```

> By default, the SAP Cloud SDK uses `[axios](https://github.com/axios/axios)` as an HTTP client backend for executing requests.

Reviewer 2: ```suggestion
By default, the SAP Cloud SDK uses `[axios](https://github.com/axios/axios)` as an HTTP client for executing requests.
```

The SAP Cloud SDK derives and configures most of the request options like `url`, `headers`, and others for you.
In certain cases passing additional options to `axios` might be required.

> To achieve this you can provide a custom request configuration that will be passed down to `axios` and in turn, applied to your requests down with the SAP Cloud SDK.

Reviewer 2: ```suggestion
To achieve this you can provide a custom request configuration that will be passed down to `axios`.
```
